dotcom-rendering/src/components/AdSlot.web.tsx

```diff
@@ -261,9 +263,12 @@ const frontsBannerAdTopContainerStyles = css`
 	min-height: ${frontsBannerMinHeight}px;
 	background-color: ${palette.neutral[97]};
 
-	${until.desktop} {
+	${until.tablet} {
```

Comment:
Could we use `from` instead of `until` here too? I think it would read a bit better. We don't want to set a min-height until the tablet breakpoint. Then at the desktop breakpoint, we want to update the min-height. We'd have to reverse `display` as well

```suggestion
	display: none;
	${from.tablet} {
	    display: flex;
		min-height: ${frontsBannerMinHeightTablet}px;
	}
	${from.desktop} {
		min-height: ${frontsBannerMinHeight}px;
	}
```